### ISSUE TITLE
Editable support

### ISSUE THREAD

**Issue by ramiroaisen** [metonym/svelte-highlight#105]
It would be nice if we could make the Highlight editable.
Something like <Highlight editable={true} {code} {language}/>
Any ideas of how can it be done?
I think with something like contentEditable attr

**Comment by metonym** [comment]
I think this would be a great feature – will explore this.

**Comment by metonym** [comment]
@ramiroaisen Editing support added in #106 and released in v0.5.0 using the `contenteditable` prop.

Sample usage:

```html
<script>
  import Highlight from 'svelte-highlight';
  import { typescript } from 'svelte-highlight/languages';
  import 'svelte-highlight/styles/github.css';

  $: code = 'let count: number = 0';
</script>

<Highlight contenteditable language={typescript} bind:code />

{code}
```

[Demo in Svelte REPL](https://svelte.dev/repl/8303de9310f94d15983ab586a0e3d256?version=3.16.7)

In the demo, edit the highlighted code block. On blurring, the bound `code` prop will update.
